src/middleware.js

```diff
@@ -29,7 +29,8 @@ export const createPolyglotMiddleware = (catchedAction, getLocale, getPhrases) =
         return next => action => {
             if (actions.includes(action.type)) {
                 const locale = getLocale(action);
-                getPhrases(locale).then(phrases => {
+                next(action);
```

Comment:
```js
                const locale = getLocale(action);
                const nexted = next(action);
                return getPhrases(locale).then(phrases => {
                    dispatch(setLanguage(locale, phrases));
                    return nexted;
                });

```

Reply:
This will help to have asynchronous chaining between middlewares : 
- The middleware before the polyglot middleware can `await next(action)`
- The middleware after the polyglot middleware can return a promise in order to add asynchronous operations to the `getPhrases(...)` promise